src/flint/fmpq_mat.jl

```diff
@@ -1039,3 +1058,19 @@ function nullspace(A::QQMatrix)
 
   return nullity, NQQ
 end
+
+################################################################################
+#
+#  Convert a matrix to array
+#
+################################################################################
+
+function to_array(M::QQMatrix)
+  A = Vector{QQFieldElem}(undef, ncols(M) * nrows(M))
+  for i = 1:nrows(M)
+    for j = 1:ncols(M)
+      A[(i - 1) * ncols(M) + j] = M[i, j]
+    end
+  end
+  return A
+end
```

Comment:
Does anybody need this function? My impression is that it is not used in our packages and it is not exported.

Reply:
We have `_vec` in Oscar which does basically the same, only worse. Maybe we can unify these two and move to AA?

https://github.com/oscar-system/Oscar.jl/blob/0b282fbd8fe07b00a6dbfc4fc077193ab29f55b3/src/Modules/missing_functionality.jl#L92

Reply:
I would prefer to remove this function. But it doesn't have to be in this PR.

Reply:
If we agree that it can go, I would just do it here.

Reply:
If it is not used in Hecke/Oscar, just remove it.